### PR TITLE
Updated TEI and image OAI feed to match public site display assumptions

### PR DESCRIPTION
**Pull request by kjallen**
This PR addresses: 
- [SYS-1574](https://uclalibrary.atlassian.net/browse/SYS-1574) Image shows up in download options
- [SYS-1575](https://uclalibrary.atlassian.net/browse/SYS-1575) XML transcript does not appear in individual item display

Once the reason was hunted down the code changes were minimal. The label was changed for the interviewee image, which was triggering the download. The new label no longer includes `Narrator` which triggers the download.

The TEI xml transcript label was updated from `display label` to `timed log`, and an extra check was added to only include xml documents.

To test:
- Run tests via `docker compose exec django python manage.py test`, all should pass. Tests previously existed for transcripts and image, but the label check was corrected
- View [an item](http://localhost:8000/oai/?verb=GetRecord&identifier=21198/zz002kppdz) with an image. The following text should be present:
`<mods:location displayLabel="Image of Interviewee">
<mods:url>https://static.library.ucla.edu/oralhistory/submasters/21198-zz002kppdz-2-submaster.jpg</mods:url>
</mods:location>`
- View [an item](http://localhost:8000/oai/?verb=GetRecord&identifier=21198/zz00096c5n) with TEI transcripts (the timed log). The following text should be present:
```
<mods:relatedItem xlink:href="https://wowza.library.ucla.edu/dlp/definst/mp3:oralhistory/21198-zz0009d33f-1-submaster.mp3/playlist.m3u8" type="constituent">

<mods:titleInfo>

<mods:title>SESSION 1 (3/10/2008)</mods:title>

</mods:titleInfo>

<mods:identifier>21198/zz0009d33f</mods:identifier>

<mods:part order="1" type="session_audio"/>
...

<mods:location>
<mods:url usage="timed log">https://static.library.ucla.edu/oralhistory/text/submasters/21198-zz0009d33f-2-submaster.xml</mods:url>
</mods:location>


</mods:relatedItem>
```

[SYS-1574]: https://uclalibrary.atlassian.net/browse/SYS-1574?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SYS-1575]: https://uclalibrary.atlassian.net/browse/SYS-1575?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: oh_staff_ui/classes/OralHistoryMods.py [hunk 1]
```diff
@@ -191,7 +191,7 @@ def _populate_narrator_image(self):
             item=self._item, file_type__file_code="image_submaster"
         ).order_by("sequence"):
             self.locations.append(
-                LocationOH(url=img.file_url, label="Image of Narrator")
+                LocationOH(url=img.file_url, label="Image of Interviewee")
```
Comment (akohler) [inline oh_staff_ui/classes/OralHistoryMods.py:194]: This change of label from Narrator to Interviewee is fine, but it's in a method called `_populate_narrator_image()`.  The method name should change as well, to reflect its purpose.

File: oh_staff_ui/classes/OralHistoryMods.py [hunk 2]
```diff
@@ -177,12 +177,12 @@ def _create_relateditem_audio(self, mi: MediaFile) -> MODSv34:
...
 
         for ts in MediaFile.objects.filter(
-            item=pi, file_type__file_code="text_master_index"
+            item=pi, file_type__file_code="text_master_transcript"
         ):
-            if ts.file_url != "":
-                ri.locations.append(
-                    LocationOH(url=ts.file_url, usage="primary display")
-                )
+            if ts.file_url != "" and ts.file_url.endswith(".xml"):
```
Comment (akohler) [inline oh_staff_ui/classes/OralHistoryMods.py:182]: Please add a test to cover this.  We already have `test_timed_log_attribute_is_added()` which covers the `text_master_transcript` case, but should have a new test to confirm that `text_master_index` files do *not* cause the addition of the timed log attribute.

**Review by akohler — CHANGES_REQUESTED** [review]
@kjallen These code changes are good, and I appreciate the comment with context.  A couple of changes - a method name and a test - and this can be merged.

It will also need our standard changes for [deployment](https://uclalibrary.atlassian.net/wiki/x/OYAwDw): bumping the production values image tag value and brief bug fix release notes.

Thanks --Andy

File: oh_staff_ui/classes/OralHistoryMods.py (hunk 2 above)
Comment (kjallen) [reply oh_staff_ui/classes/OralHistoryMods.py:182]: Added `test_absence_of_timed_log_with_text_master_index` test to confirm behavior when **only** a `text_master_index` is present

**Review by akohler — APPROVED** [review]

**Comment by akohler** [comment]
@kjallen Thanks for making the changes.  All tests pass, and I see the expected data in the OAI examples.

There's a new utility method, `ModsTestCase.prettify_xml()`, which isn't used.  Since it's in `tests.py` and it looks useful, I left it in :) - but I took out an unused import `from copy import deepcopy`, and wrapped a couple of long comments (more than 100 characters) which flake8 was complaining about.

Merged.

**Pull request merged**